x-pack/plugins/ml/public/timeseriesexplorer/timeseriesexplorer_controller.js

```diff
@@ -447,6 +454,33 @@ module.controller('MlTimeSeriesExplorerController', function (
       console.log('Time series explorer - error getting scheduled events from elasticsearch:', resp);
     });
 
+    // Query 4 - load any annotations for the selected job.
+    if (mlAnnotationsEnabled) {
+      ml.annotations.getAnnotations({
+        jobIds: [$scope.selectedJob.job_id],
+        earliestMs: searchBounds.min.valueOf(),
+        latestMs: searchBounds.max.valueOf(),
+        maxAnnotations: DEFAULT_QUERY_SIZE
+      }).then((resp) => {
+        refreshFocusData.focusAnnotationData = resp.annotations[$scope.selectedJob.job_id]
+          .sort((a, b) => {
+            return a.timestamp - b.timestamp;
+          })
+          .map((d, i) => {
+            d.key = String.fromCharCode(65 + i);
+            return d;
+          });
+
+        finish();
+      }).catch(() => {
+        // silent fail
```

Comment:
Nit: Might be good to log something here so that we know where something went wrong. 🤔

Reply:
Took a note to improve that in a follow-up. Problem with displaying e.g. a toast just like this is that it would show up every time if there are no annotations because it passes on a `404`.